nasa/earthdata-search
# EDSC-4140: convert cypress retrievals/downloads.cy.js to playwright.

# Overview

### What is the feature?

Convert cypress retrievals/downloads.cy.js to playwright.

### What is the Solution?

Pretty straight forward. Just reproduced the cypress test in plawright.

### What areas of the application does this impact?

playwright test

# Testing

### Reproduction steps

- **Environment for testing:**localhost
- **Collection to test with:**n/a

1. run the playwright tests using ```npm run playwright:ui```
2. expand the download tab until you hit the actual test and click the green play button and run the test.
3. Run through the rest of the playwright tests and make sure nothing else breaks.


### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.
<img width="1320" alt="image" src="https://github.com/nasa/earthdata-search/assets/8591291/7cb9f2f2-f393-47cc-81eb-1a373f593dca">


# Checklist
